src/index.js

```diff
@@ -14,11 +14,15 @@ const GitUtils = require('./GitUtils.js');
 const HelixConfig = require('./HelixConfig.js');
 const Strain = require('./Strain.js');
 const Logger = require('./Logger.js');
+const sequence = require('./sequence.js');
+const dom = require('./dom.js');
 
 module.exports = {
   GitUrl,
   GitUtils,
   HelixConfig,
   Strain,
   Logger,
+  ...sequence,
```

Comment:
hmm. we're tossing here quite some random utilities into the toplevel export. maybe better to use `Sequence` as container. 

```suggestion
  Sequence: sequence,
```

Reply:
uppercase doesn't seem right since it's not an object. I'd be ok with lower case…

Reply:
lowercase is ok. it's a module-ish object anyways.